pyannotate_tools/fixes/fix_annotate_json.py

```diff
@@ -48,7 +48,11 @@ def crawl_up(arg):
     """
     dir, mod = os.path.split(arg)
     mod = strip_py(mod) or mod
+    cwd = os.getcwd()
     while dir and get_init_file(dir):
+        if cwd == dir:
```

Comment:
To make the comparison more likely to succeed, you can use `if cwd == os.path.realpath(dir):`.